index.js

```diff
@@ -448,8 +450,11 @@ const vm = new Vue({
 			})
 
 			//only keep 16 color profiles
+			//if (this.colorProfilesMainColors.length > this.MAX_ALT_PALETTES)
+			//	this.colorProfilesMainColors.splice(this.MAX_ALT_PALETTES, this.colorProfilesMainColors.length - 1)
+
 			if (this.colorProfilesMainColors.length > this.MAX_ALT_PALETTES)
-				this.colorProfilesMainColors.splice(this.MAX_ALT_PALETTES, this.colorProfilesMainColors.length - 1)
+				this.MAX_ALT_PALETTES = this.colorProfilesMainColors.length; //prefer maintaining user data.

```

Comment:
also not sure this comment's really useful in the context of a comment in the code - since it says "prefer" it refers to the change you've made in the commit, but a better place to do that would be to do it in the PR or in the commit message (and you did both already :muscle:)

(sorry, that's kinda nitpicking haha, but since I'm already asking to just clean up the commented code, I might as well!!)